src/components/media/MediasSelectModal.jsx

```diff
@@ -291,16 +338,290 @@ const MediasSelectModal = ({
                 />
               </Center>
             )}
-            <LoadingButton
-              alignSelf="flex-end"
-              id="selectMedia"
-              isDisabled={!watch("selectedMedia")}
-              onClick={handleSubmit((data) => onProceed(data))}
-            >
-              Select
-            </LoadingButton>
-          </VStack>
-        </ModalFooter>
+          </div>
+        </Box>
+      </GridItem>
+    </Grid>
+  )
+
+  const FolderBreadcrumb = () => (
+    <VStack>
+      <Breadcrumb alignSelf="start">
+        {queryParams.mediaDirectoryName
+          ? queryParams.mediaDirectoryName.split("%2F").map((dir, idx) => (
+              <BreadcrumbItem
+                isCurrentPage={
+                  idx === queryParams.mediaDirectoryName.split("%2F").length - 1
+                }
+              >
+                <BreadcrumbLink
+                  onClick={(e) => {
+                    e.preventDefault()
+                    setQueryParams((prevState) => ({
+                      ...prevState,
+                      mediaDirectoryName: getMediaDirectoryName(
+                        queryParams.mediaDirectoryName,
+                        { end: idx + 1 }
```

Comment:
could i check why the end parameter is different here? previously it was `length - 1`

Reply:
`idx` is the existing position of the element already (0-indexed)! So the stop point is one after this one